region/brinstar/red/Red Tower.json

```diff
@@ -2285,16 +2285,25 @@
       "link": [7, 6],
       "name": "SpringBall Jump",
       "requires": [
-        "canSpringBallJumpMidAir",
+        "canTrickySpringBallJump",
         {"or": [
-          "canWalljump",
+          "canConsecutiveWalljump",
+          "HiJump",
           {"and": [
-            "HiJump",
-            "canTrickyJump"
-          ]}
+            "canTrickyJump",
+            "canLateralMidAirMorph",
+            "canResetFallSpeed",
+            {"or": [
+              {"ammo": {"type": "PowerBomb", "count": 1}},
+              {"ammo": {"type": "Super", "count": 1}},
+              {"enemyDamage": {"enemy": "Ripper", "type": "contact", "hits": 1}}
```

Comment:
```suggestion
              {"enemyDamage": {"enemy": "Ripper", "type": "contact", "hits": 1}},
              "canTrickyDodgeEnemies"
```
It's not important, but a "canTrickyDodgeEnemies" can cover variations like what is done in the video, as an alternative to killing the top Ripper or doing a spring fling.